lnrpc/rpc_utils.go

```diff
@@ -193,3 +196,41 @@ func GetChanPointFundingTxid(chanPoint *ChannelPoint) (*chainhash.Hash, error) {
 
 	return chainhash.NewHash(txid)
 }
+
+// CalculateFeeRate uses either satPerByte or satPerVByte, but not both, from a
```

Comment:
> However, other routines, such as FundPsbt inside
of walletkit_server.go, could also rely on the same fee calculation
functionality.

Is there a followup PR to make such change?

Reply:
Cool yeah it's up to you. This is non-blocking for this specific PR since we understand there's a follow-up PR. But for future PRs it's better to make minimal changes in a single PR so it gets approved faster🤓